block/pkg/datastore/init.go

```diff
@@ -0,0 +1,5 @@
+package datastore
```

Comment:
What is the purpose of this Blank file?

Reply:
This is for a purpose as explained above